tests/app/main/views/test_security_policy.py

```diff
@@ -0,0 +1,8 @@
+def test_security_redirects_to_policy(
+    client_request
+):
+    client_request.get(
+        'main.security_policy',
```

Comment:
For endpoints with a ‘standard’ URL I think it’s better to test the path itself which gives us the extra confidence that the `main.route` lines are correct:
```suggestion
    client_request.get_url(
        '/security.txt',
```

You’d probably want to parametrize the text to cover both of the two paths as well.

There’s precedent with `robots.txt` here:
https://github.com/alphagov/notifications-admin/blob/787cb3ef1f525daef3191d48fa87290aa1cd7126/tests/app/main/views/test_index.py#L63-L64

Reply:
Sensible, have rebased this change in. Should be good for a second review please